service/web/tabs/app/src/routes/index.tsx

```diff
@@ -11,45 +10,40 @@ import {
   ServiceVariantsContainer,
   LayoutContainer
 } from "../containers"
-import { RESERVED_VARIANT } from "../utilities"
 
 const routes = (
   <div>
     <Switch>
-      {/* Maintain compatibility for services that do not need variant functionality */}
       <Route
-        path="/app/services/:service/create"
-        component={CreateFormContainer}
+        path="/app/services/:service/variants/"
+        component={ServiceVariantsContainer}
       />
       <Route
         path="/app/services/:service/variants/:variant/create"
         component={CreateFormContainer}
       />
       {/* Maintain compatibility for services that do not need variant functionality */}
       <Route
-        path="/app/services/:service/runs/:offset"
-        component={ServiceRunsContainer}
+        path="/app/services/:service/create"
+        component={CreateFormContainer}
       />
       <Route
         path="/app/services/:service/variants/:variant/runs/:offset"
         component={ServiceRunsContainer}
       />
       {/* Maintain compatibility for services that do not need variant functionality */}
       <Route
-        path="/app/services/:service"
-        component={ServiceDetailsContainer}
+        path="/app/services/:service/runs/:offset"
+        component={ServiceRunsContainer}
       />
       <Route
-        path="/app/services/:service/variants/:variant"
-        component={ServiceDetailsContainer}
-      />
-      <Redirect
```

Comment:
This no longer should be a redirect?

Reply:
correct. This shouldn't have been there.